pkgs/development/beam-modules/default.nix

```diff
@@ -44,27 +44,27 @@ let
       # BEAM-based languages.
       elixir = elixir_1_13;
 
-      elixir_1_13 = lib'.callElixir ../interpreters/elixir/1.13.nix {
+      elixir_1_14 = lib'.callElixir ../interpreters/elixir/1.14.nix {
```

Comment:
Look how the diff ended up. Maybe instead of adding to the top could add to the bottom.